metashare/api/sf_org_changes.py

```diff
@@ -44,21 +54,21 @@ def get_valid_target_directories(scratch_org_config):
 
     if os.path.isdir("unpackaged/pre"):
         package_directories["pre"] = [
-            dirname
+            "unpackaged/pre/" + dirname
             for dirname in os.listdir("unpackaged/pre")
-            if os.path.isdir(dirname)
+            if os.path.isdir("unpackaged/pre/" + dirname)
         ]
     if os.path.isdir("unpackaged/post"):
         package_directories["post"] = [
-            dirname
+            "unpackaged/post/" + dirname
             for dirname in os.listdir("unpackaged/post")
-            if os.path.isdir(dirname)
+            if os.path.isdir("unpackaged/post/" + dirname)
         ]
     if os.path.isdir("unpackaged/config"):
         package_directories["config"] = [
-            dirname
+            "unpackaged/config/" + dirname
             for dirname in os.listdir("unpackaged/config")
-            if os.path.isdir(dirname)
+            if os.path.isdir("unpackaged/config/" + dirname)
```

Comment:
@wlonk These changes ensure that we're storing the full path, not just the final dirname. Feel free to cleanup as needed.

Reply:
I think they're fine as-is.